app/services/migration/migrators/application.rb

```diff
@@ -105,6 +108,16 @@ def provider_only_in_npq?(application, ecf_npq_application)
         ecf_npq_application.private_childcare_provider_urn.nil? && application.private_childcare_provider_including_disabled.present?
     end
 
+    def touch_updated_at?(application, ecf_npq_application)
+      ecf_npq_application.profile&.school_urn.presence != ecf_npq_application.school_urn ||
+        application.ineligible_for_funding_reason != ecf_npq_application.ineligible_for_funding_reason ||
```

Comment:
I don't think this will work properly, as we might not have all the applications from the migrator done yet as the calculation is expected on all applications for that user. Lets leave ineligible for funding out for now
I'll rethink how we add it in again